README.md

```diff
@@ -12,6 +12,18 @@ https://developer.webex.com/docs/login-with-webex
 
 Try it out by chosing one of the available flows: https://webexsamples.github.io/login-with-webex
 
+To start the server run:
+
+`npm start`
+
+Optional parameters are:
+WEBEX_BASE=https://integration.webexapis.com
```

Comment:
I'd format these guys as an unordered list and enclose the param names in backticks.

Reply:
thanks for the review - updated!